src/main/java/uk/ac/ebi/embl/converter/gff3toff/GFF3Mapper.java

```diff
@@ -89,12 +90,30 @@ private Feature mapGFF3Feature(GFF3Feature gff3Feature) {
         Map<String, String> attributes = gff3Feature.getAttributes();
         Collection<Qualifier> qualifiers = mapGFF3Attributes(attributes);
 
-        CompoundLocation<Location> locations = mapGFF3Location(gff3Feature);
 
         String featureType = gff3Feature.getName();
-        Feature ffFeature = this.featureFactory.createFeature(featureType);
-        ffFeature.setLocations(locations);
-        ffFeature.addQualifiers(qualifiers);
+
+        String featureHashId;
+         if (attributes.containsKey("ID")) {
+             featureHashId = attributes.get("ID");
+         } else {
+             featureHashId = gff3Feature.hashCode() + "";
+         }
```

Comment:
This could be simplified 
`String featureHashId = attributes.getOrDefault("ID", String.valueOf(gff3Feature.hashCode()));`